projects/core/src/configurator/commons/store/effects/configurator.effect.ts

```diff
@@ -352,17 +357,18 @@ export class ConfiguratorEffects {
 
   getGroupWithAttributes(groups: Configurator.Group[]): string {
     const groupWithAttributes: Configurator.Group = groups
-      .filter(currentGroup => currentGroup.attributes)
+      .filter(currentGroup => currentGroup.attributes.length > 0)
```

Comment:
This assumes that currentGroup.attributes is always defined, right? Why is currentGroup.attributes not enough?

Reply:
Attributes is always defined from the normalizer, so checking only if attributes is existing is not sufficient.